### PR TITLE
feat(workflow): Team alerts triggered graph

### PR DESCRIPTION
**Pull request by scttcper**
Makes an api request and displays a graph

look at this interesting fake data
Screen Shot 2021-09-24 at 4 41 52 PM

### CODE REVIEW

**Review by 0ex-d — APPROVED** [review]
Sleek tests

File: static/app/views/teamInsights/teamAlertsTriggered.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, start, end, period, utc, teamSlug} = this.props;
+    const datetime = {start, end, period, utc};
+
+    return [
+      [
+        'alertsTriggered',
+        `/organizations/${organization.slug}/${teamSlug}/alerts-triggered/`,
```
Comment (matejminar) [inline static/app/views/teamInsights/teamAlertsTriggered.tsx:41]: Does endpoint like this already exist?

**Review by matejminar — APPROVED** [review]
Other than that endpoint, lgtm.

File: static/app/views/teamInsights/teamAlertsTriggered.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/views/teamInsights/teamAlertsTriggered.tsx:41]: Coming in #28816

**Pull request merged**